1.Core/Helpers/APIRestHelper.cs

```diff
@@ -29,7 +29,13 @@ public RestResponse DoRequest(Method method, string endpoint, string? body)
             request.AddParameter("application/json", body, ParameterType.RequestBody);
         }
 
-        RestResponse response = client.Execute(request);
-        return response;
+        try
+        {
+            return client.Execute(request);
+        }
+        catch (Exception e)
```

Comment:
It is not a good practice to catch an exception with the type "Exception", it is too general, it is better to catch a more specific type of exception.

Reply:
My mistake, fixed